medusa/tv/cache.py

```diff
@@ -124,9 +106,9 @@ def __init__(self, provider_id):
                     b'    time NUMERIC)'
                 )
         except Exception as error:
-            logger.log('Error while searching {provider_id}, skipping: {e!r}'.
-                       format(provider_id=provider_id, e=error), logger.DEBUG)
-            logger.log(traceback.format_exc(), logger.DEBUG)
+            log.debug('Error while searching {provider_id}, skipping: {error!r}',
+                      {'provider_id': provider_id, 'error': error})
+            log.debug(traceback.format_exc())
```

Comment:
Actually thinking about it I would probably stick with `traceback.format_exc()` but I would also still combine them.  `traceback.format_exc()` prints out the `exc_info` formatted and since we have no need to change the formatting (such as stripping linebreaks, etc) its probably the best option.